### ISSUE TITLE
Wrong DummyRendering path

### ISSUE THREAD

**Issue by stat1x** [helhum/typoscript_rendering#42]
Hi,
I've got the following warning when calling an action with typoscript_rendering :
```
PHP Warning: include(typo3conf/ext/typoscript_rendering/Scripts/DummyRendering.php): failed to open stream: No such file or directory in /pathToMySite/vendor/typo3/cms/typo3/sysext/frontend/Classes/Http/RequestHandler.php line 206 (More information)
```
I think it is due to wrong DummyRendering path on line 46 of `FrontendRenderingProvider.php`.

**Comment by helhum** [comment]
> I think it is due to wrong DummyRendering path on line 46 of 

Why do you think this is a wrong path? Path looks correct to me. 

What TYPO3 version are you using? What PHP version? Any base dir restriction or similar?

**Comment by stat1x** [comment]
Hi,
I'm using TYPO3 8.7.9 with PHP 7.1. I don't have any base dir restrinction. If I add a `/` like this I don't get the warning : 
`/typo3conf/ext/typoscript_rendering/Scripts/DummyRendering.php`

**Comment by helhum** [comment]
> If I add a /

can you outline where exactly you add this?

In the code I have this:

https://github.com/helhum/typoscript_rendering/blob/8cd6083ed025aa566a89eac58d85f44038e9007f/Classes/Core/FrontendRenderingProvisioner.php#L46

**Comment by helhum** [comment]
and can you check if it works with 8.7.8?

**Comment by helhum** [comment]
oh, and how do you run PHP? fastcgi, apache or fpm?

In any case, this looks like a bug in TYPO3

**Comment by stat1x** [comment]
I run it with integrated server `php -S localhost:8000`

**Comment by stat1x** [comment]
I add it like this : `$typoScriptFrontendController->config['config']['pageGenScript'] = '/typo3conf/ext/typoscript_rendering/Scripts/DummyRendering.php'; `

**Comment by helhum** [comment]
@rschaufler @stat1x can you confirm that the fix applied in mentioned commit works for you? It fixes the issue here.

**Comment by rschaufler** [comment]
@helhum i can confirm that the issue is fixed using the current master branch, thank you!

**Comment by helhum** [comment]
@rschaufler Thanks. Released as 2.1.0
